rewrite-java/src/main/java/org/openrewrite/java/JavaPrinter.java

```diff
@@ -843,6 +843,16 @@ public J visitNewClass(NewClass newClass, PrintOutputCapture<P> p) {
         return newClass;
     }
 
+    @Override
+    public J visitNullableType(J.NullableType nt, PrintOutputCapture<P> p) {
+        beforeSyntax(nt, Space.Location.UNARY_PREFIX, p);
```

Comment:
Should we use `NULLABLE_PREFIX` here?
```suggestion
        beforeSyntax(nt, Space.Location.NULLABLE_PREFIX, p);
```

Reply:
this has been fixed.